inc/osvr/ClientKit/Display.h

```diff
@@ -432,6 +437,34 @@ namespace clientkit {
             return osvrClientCheckDisplayStartup(m_disp) == OSVR_RETURN_SUCCESS;
         }
 
+        /// @brief Returns number of inputs for this display.
+        ///
+        /// @sa osvrClientGetNumDisplayInputs()
+        OSVR_DisplayInputCount getNumDisplayInputs() const {
+            ensureValid();
+            OSVR_DisplayInputCount inputs;
+            OSVR_ReturnCode ret = osvrClientGetNumDisplayInputs(m_disp, &inputs);
+            if (ret != OSVR_RETURN_SUCCESS) {
+                handleDisplayError("Couldn't get number of inputs of this "
+                                   "display!");
```

Comment:
Really minor thing - should probably say "Couldn't get number of display inputs!" Since each display input could be either inputs to a single display or multiple displays.